adcircpy/forcing/winds/best_track.py

```diff
@@ -809,3 +799,23 @@ def read_atcf(track: PathLike) -> DataFrame:
                 data[key] = value
 
     return DataFrame(data=data)
+
+
+def get_circle_of_radius(lon, lat, radius):
+
+    local_azimuthal_projection = '+proj=aeqd +R=6371000 +units=m ' f'+lat_0={lat} +lon_0={lon}'
+    wgs84_to_aeqd = partial(
+        pyproj.transform,
+        pyproj.Proj('+proj=longlat +datum=WGS84 +no_defs'),
+        pyproj.Proj(local_azimuthal_projection),
+    )
+    aeqd_to_wgs84 = partial(
+        pyproj.transform,
+        pyproj.Proj(local_azimuthal_projection),
+        pyproj.Proj('+proj=longlat +datum=WGS84 +no_defs'),
+    )
+
+    center = Point(float(lon), float(lat))
+    point_transformed = ops.transform(wgs84_to_aeqd, center)
+
+    return ops.transform(aeqd_to_wgs84, point_transformed.buffer(radius))
```

Comment:
will this fail with vertical datum changes?

Reply:
> will this fail with vertical datum changes?

No, this is completely independent of the vertical datum.

But keep in mind, one thing most people don't realize is that WGS84 implies an _ellipsoidal_ vertical datum, so categorically speaking, WGS84 is not exactly the "correct" CRS for these meshes in the vertical. It would have to be a CRS with a _geoidal_ datum.

Since we are not considering vertical transformations here anyways, WGS84 works well for horizontal referencing of meshes in lon/lat, but it's not exactly the correct referring for the topobathy (vertical) values.